Minor correction change in about-disks-and-vhds.md

bahreex opened the pull request:
The statement around using a VHD to create a data disk through Azure was bit misleading. Corrected the statement by slightly expanding the explanation.

azurecla commented:
Hi __@SSalahAhmed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

bahreex committed:
Minor correction change in about-disks-and-vhds.md

The statement around using a VHD to create a data disk through Azure was bit misleading. Corrected the statement by slightly expanding the explanation.
